util/reggen/gen_dv.py

```diff
@@ -72,6 +72,65 @@ def gen_core_file(outdir: str,
         yaml.dump(core_data, core_file, encoding='utf-8')
 
 
+def get_all_dv_base_names(dv_base_prefix):
+    ''' Return all DV base names in a dictionary.
+
+    The output format is:
+    {
+      block_1: ["base_reg_block_name", "base_reg_name", "base_reg_field_name", "base_reg_pkg"],
+      block_2: ["base_reg_block_name", "base_reg_name", "base_reg_field_name", "base_reg_pkg"]
+    }
+    '''
+    base_names = {}
+    default_prefix = "dv_base"
+    default_postfix = ["_reg_block", "_reg", "_reg_field", "_mem", "_reg_pkg"]
+    default_base_names = [default_prefix + postfix for postfix in default_postfix]
+    base_names["default"] = default_base_names
+
+    if dv_base_prefix == default_prefix:
+        return base_names
+
+    # Customerized dv_base_prefix comes with format:
+    # --dv-base-prefix "block_name_1:type:base_name,block_name_2:type:base_name"
+    block_prefixes = dv_base_prefix.split(",")
```

Comment:
Actually the `dv_base_names` input arg comes with a string. I did not change it because I believe most cases we only change one item. But please let me know if you prefer it to be a list instead :)